BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/BugsnagPerformance.cs

```diff
@@ -136,13 +137,17 @@ internal class SceneLoadSpanContainer
 
         private void Configure(PerformanceConfiguration config)
         {
+            _isSamplingProbabilityOverriden = config.IsFixedSamplingProbability;
             _networkRequestCallback = config.NetworkRequestCallback;
             _cacheManager.Configure(config);
             _persistentState.Configure(config);
             _delivery.Configure(config);
             _resourceModel.Configure(config);
             _sampler.Configure(config);
-            _pValueUpdater.Configure(config);
+            if(!_isSamplingProbabilityOverriden)
```

Comment:
Suggest avoiding a new field and just using config directly for these two uses:

```suggestion
            if(!config.IsFixedSamplingProbability)
```

Reply:
I added an IsConfigured property to the pvalue update so that we an get rid of the extra property, much cleaner 👍